### PR TITLE
#22 Schedule updated for shared links 

### PR DESCRIPTION
**Pull request by aayusharora**
https://github.com/OpenTechSummit/2016.opentechsummit.net/issues/22

### CODE REVIEW

**Comment by mariobehling** [comment]
@aayusharora Please make it easy for reviewers. Best would be, if you set up any changes on your own repo and provide a link to the improvements in future.

File: schedule/schedule.tpl [hunk 1]
```diff
@@ -114,6 +114,7 @@
                     <a class="session-link" href="#{{session_id}}">
                       <i class="fa fa-link"></i>
                     </a>
+                    <input class="inputbox" type="text" onclick="this.select()" value="https://opentechevent.net/programm/#{{session_id}}" style="display:none;">
```
Comment (SaptakS) [inline schedule/schedule.tpl:117]: @aayusharora It should be http://opentechsummit.net/ instead of https://opentechevent.net/.

Comment (aayusharora) [inline schedule/schedule.tpl:117]: @SaptakS , solved here #14

Comment (SaptakS) [inline schedule/schedule.tpl:117]: Its opentechsummit.net .. not opentechevent.net

Comment (aayusharora) [inline schedule/schedule.tpl:117]: Corrected that https://github.com/OpenTechSummit/open-event-scraper/pull/14/files

**Pull request merged**